docs/updating/2_merge_composer.md

```diff
@@ -1,36 +1,59 @@
-# 2. Merge composer.json
+# 2. Resolve conflicts
 
-## Manual merging
+## 2.1. Resolve conflicts in `composer.lock`
 
-!!! note "Sorted packages since 2.4 changes"
+Ignore the conflicts in `composer.lock`, because this file is regenerated when you execute `composer update` later.
+It is easiest to check out the version of the file from the tag and add it to the changes.
 
-    Because the 2.4 packages in `composer.json` are sorted, there will be more conflicts when updating to 2.4, but far fewer conflicts in the future. This is controlled by [sort-packages](https://getcomposer.org/doc/06-config.md#sort-packages) config in `composer.json`.
+If you get a lot of conflicts (for example, on the `doc` folder),
```

Comment:
It feels like this part deserves its own "chapter", it's not about composer.lock

Reply:
So maybe this under general "Resolve conflicts" heading, and specifics about `composer.json` in a separate section below, like they are now?